libraries/ESP8266WiFi/examples/WiFiSmartConfig/WiFiSmartConfig.ino

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Smart config example sketch
+ *
+ */
+
+#include <ESP8266WiFi.h>
+
+extern "C" {
+#include "c_types.h"
+#include "ets_sys.h"
+#include "os_type.h"
+#include "osapi.h"
+#include "mem.h"
+#include "user_interface.h"
+#include "smartconfig.h"
+#include "lwip/opt.h"
+#include "lwip/err.h"
+#include "lwip/dns.h"
+}
+
+void setup()
+{
+    Serial.begin(115200);
+
+    // clear all ssid
+    wifi_station_clear_username();
```

Comment:
i think the code is useful. if i don't clear the ssid, i can't change ssid because the wifi auto connect to old ssid.